snorkel/labeling/model/sparse_label_model/base_sparse_label_model.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+"""Sparse Data Helpers.
+
+Indexing throughout this module is 0 based, with the assumption that "abstains" are ommited.
+
+When working with larger datasets, it can be convenient to load the data in sparse format. This module
```

Comment:
Nit: Let's move this summary of this file's purpose above the indexing disclaimer.